src/couch/entry.js

```diff
@@ -225,7 +227,9 @@ const methods = {
     let action = 'updated';
     if (entry._id) {
       try {
-        const doc = await this.getEntryWithRights(entry._id, user, ['write']);
+        const doc = await this.getEntryWithRights(entry._id, user, ['write'], {
+          token: token,
```

Comment:
```suggestion
          token: options.token,
```

Isn't this enough (without the other line above) ?